rpm/libqofono-qt5.spec

```diff
@@ -67,9 +63,12 @@ export QT_SELECT=5
 make %{?_smp_mflags}
 
 %install
-rm -rf %{buildroot}
 export QT_SELECT=5
 %qmake5_install
+# MeeGo.QOfono legacy import
+mkdir -p %{buildroot}%{_libdir}/qt5/qml/MeeGo/QOfono
+ln -sf %{_libdir}/qt5/qml/QOfono/libQOfonoQtDeclarative.so %{buildroot}%{_libdir}/qt5/qml/MeeGo/QOfono/
```

Comment:
This might be better if it was relative. This seems to annoy our SDK:
```
Resetting snapshot 'SailfishOS-devel-armv7hl.default' of 'SailfishOS-devel-armv7hl' target...
cp: not writing through dangling symlink '/srv/mer/targets/SailfishOS-devel-armv7hl.default/usr/lib/qt5/qml/MeeGo/QOfono/libQOfonoQtDeclarative.so'
Removing destination and trying again...
```

Reply:
Sure. Done.